doc/sphinx/src/models.rst

```diff
@@ -1277,10 +1277,13 @@ average atomic mass ``Abar`` and atomic number ``Zbar`` for heavy
 ions, assuming nuclear statistical equilibrium.
 
 In addition, the user may query the bounds of the table via the
-functions ``lRhoMin()``, ``lRhoMax()``, ``lTMin()``, ``lTMax()``,
-``TMin()``, ``TMax()``, ``YeMin()``, ``YeMax()``, ``sieMin()``, and
-``sieMax()``, which all return a ``Real`` number. The ``l`` prefix
-indicates log base 10.
+functions ``rhoMin()``, ``rhoMax()``, ``TMin()``, ``TMax()``,
+``YeMin()``, ``YeMax()``, ``sieMin()``, and ``sieMax()``, which all
+return a ``Real`` number.
+
+.. warning::
+    As with the SpinerEOS models, the stellar collapse models use fast
+    logs.
```

Comment:
Is fast log a configurable compile time option? If so should that option be documented here?

Reply:
It is. I'll add a comment to that effect.